lib/okta.go

```diff
@@ -67,7 +68,7 @@ func (c *OktaCreds) Validate() error {
 	return nil
 }
 
-func NewOktaClient(creds OktaCreds, oktaAwsSAMLUrl string, sessionCookie string) (*OktaClient, error) {
+func NewOktaClient(creds OktaCreds, oktaAwsSAMLUrl string, sessionCookie string, mfaConfig MFAConfig) (*OktaClient, error) {
```

Comment:
This breaks the API. We're still v0, so I guess that's fine.

Reply:
Would it be better to not pass it to `NewOktaClient` and just set it with `client.MFAConfig = config` when/where you need it set?  (eg in `OktaProvider.Retrieve()`)

Reply:
Sure. Or a new constructor like `NewOktaClientWithMFAConfig`

Reply:
Regarding your edit, that would be fine. I think asking the caller to add it after the fact isn't the worst.

Re `OKTA_MFA_ID`: I'd rather not have the lib pulling from env vars, seems leaky to me. But in cmd, sure.